images/goland/Dockerfile.centos

```diff
@@ -1,23 +1,7 @@
-FROM centos:8
+FROM codercom/enterprise-multieditor:centos
 
-RUN yum update -y && yum install -y \
-    which \
-    git \
-    bash \
-    curl \
-    man \
-    vim \
-    sudo \
-    gnupg \
```

Comment:
@nathanpotter did we need gnupg?

Reply:
I don't think we needed it, I think it was if you wanted to add a gpg key for signing your commits, but we can prob leave it out

Reply:
Cool, I'll leave it out for now since it wasn't in the RFC but it's trivial to add if we have people running into that.